dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.2.3
```

Comment:
Good question although this seems to be an artifact from an ealier PR and should go away with a rebase because the none shopify version of `2.2.3` is what's currently in [master](https://github.com/Shopify/shipit-engine/blob/master/dev.yml#L10)

I'll rebase before 🚢